integration_tests/ethereum/contracts/MockAaveV2StablecoinCellar.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./interfaces.sol";
+
+contract MockAaveV2StablecoinCellar is Ownable {
+    /**
+     * @notice Whether or not the contract is paused in case of an emergency.
+     */
+    bool public isPaused;
+
+    constructor() {
+
+    }
+
+    event mockAccrueFees();
+
+    /**
+     * @notice Take platform fees and performance fees off of cellar's active assets.
+     */
+    function accrueFees() external {
+        emit mockAccrueFees();
+    }
+
+    event mockTransferFees();
+
+    /**
+     * @notice Transfer accrued fees to Cosmos to distribute.
+     */
+    function transferFees() external  {
```

Comment:
A lot of functions (e.g. this one) are defined in the actual contract as `onlyOwner`. Is this going to have a material impact on the test conditions?

Reply:
`onlyOwner` simply prevents processing of calls from addresses that aren't defined as `_owner` from making calls. It has no bearing on the testing in the mock contract currently.

Reply:
Is the reason we didn't include it because there isn't a stable owner address?

Reply:
I don't think it was a conscious decision, I must have removed it by mistake

Reply:
There actually should be because we are changing the owner in the hardhat config. I'll re-add and try it out.